### DISCUSSION TITLE
Python packaging scripts

### DISCUSSION THREAD

**Discussion by metalgearsloth** [space-wizards/space-station-14#21501] [Code announcements]
The python packaging scripts Tools/package_server_build.py and Tools/package_client_build.py have been replaced by running Content.Packaging. Args are similar apart from --platform being specified differently.

**Comment by zarlo** [comment]
so are you saying we should remove the python scripts?

**Reply by metalgearsloth** [reply]
Yeah, they've been intended to be replaced by Content.Packaging for a while. The only reason they're in the repo at this point is due to the watchdog which will be updated soon. By engine 177.0.0 the python scripts won't work with engine as is.

**Comment by joemar25** [comment]


**Reply by PJB3005** [reply]
Did you have ChatGPT write this comment?
